src/xcode/ENA/ENA/Source/Scenes/ContactDiary/Store/ContactDiaryStoreV1.swift

```diff
@@ -50,10 +50,48 @@ class ContactDiaryStoreV1: DiaryStoring, DiaryProviding {
 		guard case .success = updateDiaryResult else {
 			return nil
 		}
+		
+		migrateColumns(inTable: "ContactPerson")
+		migrateColumns(inTable: "Location")
 
 		registerToDidBecomeActiveNotification()
 	}
-
+	
+	func migrateColumns(inTable tableName: String) {
```

Comment:
Can we please create the migration similar to the DownloadedPackageStore?
This would mean creating: ContactDiaryStoreV1 a Migration that implements the `Migration` Protocol